tests/z3/z3_test.cpp

```diff
@@ -68,7 +68,7 @@ int main()
   cout << concat << endl;
 
   Sort bvsort2 = s->make_sort(BV, 8);
-  Term bvterm2 = s->make_term("x", bvsort2);
+  Term bvterm2 = s->make_symbol("x", bvsort2);
```

Comment:
maybe make sure this change isn't needed in other parts of the z3-specific tests?

Reply:
They run so this should be the only place.